sdk/playwrighttesting/microsoft-playwright-testing/src/utils/serviceClient.ts

```diff
@@ -144,11 +152,18 @@ export class ServiceClient {
     if (response.status === 200) {
       return JSON.parse(response.bodyAsText!) as StorageUri;
     } else {
-      throw new Error(`Received status ${response.status} from service from GET StorageUri call.`);
+      this.handleErrorResponse(response, 'getStorageUri');
     }
+    throw new Error(`Received status ${response.status} from service from GET StorageUri call.`);
   }
 
   private getServiceEndpoint(): string {
     return process.env["PLAYWRIGHT_SERVICE_REPORTING_URL"]!;
   }
+
+  private handleErrorResponse(response: PipelineResponse, action: string) {
+    if (response.status === 400 || response.status === 401 || response.status === 403) {
+      process.stdout.write(`\n${Constants.ERROR_MESSAGE[response.status].action}`);
```

Comment:
are we printing the error message during the test run? 2 things
1. Some playwright reporters flush the stdout when running the tests, during these runs the message will get lost.
2. Customers having a large test suite, will miss these messages which will come in between their runs. Better approach would be to print it at test end.

Reply:
@pratiksharma23